.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style = space
+indent_size = 2
+
```

Comment:
Adding this potentially creates confusion about what is expected in terms of formatting, and the coding guidelines call for clang-format (for which there's a .clang-format file in the root directory).

Please apply the clang-formatting to your changes.